### DISCUSSION TITLE
Fidi AuthenticationException after deleting initial Firefly account and creating a new account/PAT

### DISCUSSION THREAD

**Discussion by itswerkin** [firefly-iii/firefly-iii#6660] [Firefly III]
Hello,

I setup fireflyiii/fidi using docker over the weekend, setup an account and got fidi working.  I ran into issues with importing the CSV files my bank provides, initially no income was imported, after starting over and changing my mapping choices I was able to import income as well however my expenses were doubled and my net worth was very incorrect.  I went ahead and deleted my user account in firefly and registered again to take another swing at importing transactions and now I can't auth at all, here's what I did:

I created a new PAT, stopped both firefly and fidi containers, update .fidi.env to reflect the new PAT and started both containers again.  I have not been able to get fidi to authenticate with firefly after deleting my original account, I receive "Status code is 401. Body is: {"message":"Unauthenticated","exception":"AuthenticationException"}" on every auth attempt.  I exec'd into the db container, connected to the mysql db and inspected the personal_access_tokens table, every PAT I had created was present, all were revoked except the most recent.  On a whim I deleted the contents of this table and created another PAT, after restarting containers I still remain unable to authenticate.

I'm at a loss, I'd love to complete my eval of firefly but am blocked, please advise.

**Comment by JC5** [comment]
The authentication details are stored in your cookies. Did you press "Reauthenticate" or "Start over"?

**Comment by itswerkin** [comment]
Yes, I pressed both repeatedly...  I even launched an incognito window, I
keep getting this auth exception.

**Reply by JC5** [reply]
Then it can only be a problem with the PAT. Are you using Portainer?

**Comment by itswerkin** [comment]
I have portainer running but am not using it.

**Reply by JC5** [reply]
Sorry, then it can only be a wrong PAT.
